diagnostic.md

```diff
@@ -150,6 +183,7 @@ div {
 ```
 
 <!-- your answer starts here -->
+the divs will be ordered one after the other left to right.
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).